docs/4-software-development-practices/4.3.3-github.md

```diff
@@ -14,13 +14,13 @@ After a pull request is created in GitHub it can be reviewed by viewing a side-b
 
 ### Peer Reviews: A Sacred Space for Developers
 
-![Peer Previews](../img/git-pr.webp ':class=img-center')
+![Peer Previews](/img/git-pr.webp ':class=img-center')
```

Comment:
Will work for markdown but docsify still breaks based on the relative path it references the image from

Reply:
changed completed